### PR TITLE
3723 - 0

### PR DESCRIPTION
**Pull request by Hawatz**
## Номер
3723

## Номер задания
0

## Ссылка на видео с демо
[Link](https://gfycat.com/ru/cookeddaringladybird)

## Комментарии
Пофиксил ошибки рубокопа руками где смог, остальные добавлял в дизейбл.

### CODE REVIEW

File: 3723/0/about_dice_project.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+class DiceSet
+  attr_accessor :round, :values
+
+  def initialize
+    @values = []
+    @round = round
+  end
+
+  def roll(num)
+    @values = []
```
Comment (anatoliliotych) [inline 3723/0/about_dice_project.rb]: why do you need it here? if you initialized it during creating object, you don't need to initialize it in each method.

File: 3723/0/about_dice_project.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+class DiceSet
+  attr_accessor :round, :values
+
+  def initialize
+    @values = []
+    @round = round
```
Comment (anatoliliotych) [inline 3723/0/about_dice_project.rb]: where do you use it?

**Comment by anatoliliotych** [comment]
rename last commit, it has a typo

File: 3723/0/about_dice_project.rb (hunk 1 above)
Comment (Hawatz) [reply 3723/0/about_dice_project.rb]: Fixed

File: 3723/0/about_dice_project.rb (hunk 2 above)
Comment (Hawatz) [reply 3723/0/about_dice_project.rb]: Fixed

**Comment by anatoliliotych** [comment]
please, follow the common style for all your commit messages.

**Comment by Hawatz** [comment]
@anatoliliotych Please, check my commits

File: 3723/0/about_dice_project.rb [hunk 3]
```diff
@@ -0,0 +1,68 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+class DiceSet
+  attr_accessor :values
+
+  def roll(num)
+    @values = []
+    num.times do
+      @values << rand(1..6)
```
Comment (anatoliliotych) [inline 3723/0/about_dice_project.rb]: can you do this during array init?

Comment (Hawatz) [reply 3723/0/about_dice_project.rb]: We have 'test_dice_values_should_change_between_rolls' method in this koan. So I have to do the implementation like this.

Comment (Hawatz) [reply 3723/0/about_dice_project.rb]: Fixed

**Comment by Hawatz** [comment]
@anatoliliotych Check please DiceSet method

File: 3723/0/about_dice_project.rb [hunk 4]
```diff
@@ -0,0 +1,64 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+class DiceSet
+  attr_accessor :values
+
+  def roll(num)
+    @values = Array.new(num) { Random.rand(1..6) }
```
Comment (anatoliliotych) [inline 3723/0/about_dice_project.rb]: no need in Random.

Comment (Hawatz) [reply 3723/0/about_dice_project.rb]: Fixed

**Comment by Hawatz** [comment]
@anatoliliotych Check my fixes please

**Pull request merged**